lib/useBackHandler.js

```diff
@@ -0,0 +1,22 @@
+import React, { useEffect, useState } from 'react'
+import { BackHandler } from 'react-native'
+
+export default () => {
+  const [press, setPress] = useState(false)
+
+  backPressed = () => {
+    this.props.navigation.goBack();
```

Comment:
How can you access this.props ? :)

Reply:
after you have set navigation options the prop "this.prop.navigation" is available throughout the app you can use them to navigate.

Reply:
So you assume we will always want to trigger a goBack action when the back button is pressed ? Which means: 
1 - we HAVE to use react-navigation
2 - what if we want to just do something else than going back ?

By the way, this.props does not exist there.

You eventually want to the listener callback as an argument of the hook, but the useState is useless there in my opinion, the hook should do nothing more than a useEffect.